devfile.yaml

```diff
@@ -1,26 +1,44 @@
 specVersion: 0.0.1
 name: chectl
 projects:
-  - name: chectl
+  - 
```

Comment:
That's not a blocker but I am wondering why are you adding a newline here?

Reply:
for lisibility,
here it is not a big deal, but when you have many elements, it becomes harder to read. This is personal taste :)

Reply:
Maybe you can just insert a new line between components (c.f. https://github.com/flathub/flathub/wiki/YAML-Style-Guide)? Your formatting style doesn't look standard and someone with a different "taste" may want to change it as well.